### ISSUE TITLE
iOS Flutter 2.8.1 got crash because of EXC_BAD_ACCESS KERN_INVALID_ADDRESS

### ISSUE THREAD

**Issue by BlakeWuh** [flutter/flutter#96850]
## Steps to Reproduce

1. 100% of this crash happens when app is in foreground.
2. The crash got reported to Firebase console.
3. I havn't found a way to reproduce it.

**Expected results:** 

**Actual results:** 

```
libobjc.A.dylib
objc_msgSend
Crashed：com.apple.main-thread 
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000000000010
Crashed: com.apple.main-thread
0  libobjc.A.dylib                0x2154 objc_msgSend + 20
1  Flutter                        0x32108 (missing)
2  Flutter                        0x3cc60 (missing)
3  Flutter                        0x3bc850 (missing)
4  Flutter                        0x2eb0d8 (missing)
5  Flutter                        0x2ee6a4 (missing)
6  CoreFoundation                 0x9b7a8 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 28
7  CoreFoundation                 0x9b3a8 __CFRunLoopDoTimer + 1016
8  CoreFoundation                 0x9a87c __CFRunLoopDoTimers + 324
9  CoreFoundation                 0x94bb4 __CFRunLoopRun + 1948
10 CoreFoundation                 0x93ed0 CFRunLoopRunSpecific + 572
11 GraphicsServices               0x3570 GSEventRunModal + 160
12 UIKitCore                      0xb302d0 -[UIApplication _run] + 1052
13 UIKitCore                      0xb3584c UIApplicationMain + 164
14 Runner                         0x5e9c main + 10 (main.swift:10)
15 libdyld.dylib                  0x1140 start + 4
```

<details>
  <summary>Logs</summary>

```
Crashed：com.apple.main-thread 
EXC_BAD_ACCESS KERN_INVALID_ADDRESS 0x0000000000000010
Crashed: com.apple.main-thread
0  libobjc.A.dylib                0x2154 objc_msgSend + 20
1  Flutter                        0x32108 (missing)
2  Flutter                        0x3cc60 (missing)
3  Flutter                        0x3bc850 (missing)
4  Flutter                        0x2eb0d8 (missing)
5  Flutter                        0x2ee6a4 (missing)
6  CoreFoundation                 0x9b7a8 __CFRUNLOOP_IS_CALLING_OUT_TO_A_TIMER_CALLBACK_FUNCTION__ + 28
7  CoreFoundation                 0x9b3a8 __CFRunLoopDoTimer + 1016
8  CoreFoundation                 0x9a87c __CFRunLoopDoTimers + 324
9  CoreFoundation                 0x94bb4 __CFRunLoopRun + 1948
10 CoreFoundation                 0x93ed0 CFRunLoopRunSpecific + 572
11 GraphicsServices               0x3570 GSEventRunModal + 160
12 UIKitCore                      0xb302d0 -[UIApplication _run] + 1052
13 UIKitCore                      0xb3584c UIApplicationMain + 164
14 Runner                         0x5e9c main + 10 (main.swift:10)
15 libdyld.dylib                  0x1140 start + 4
```

```
[✓] Flutter (Channel stable, 2.8.1, on macOS 11.6 20G165 darwin-arm, locale zh-Hans-CN)
    • Flutter version 2.8.1 at /Users/blake/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 77d935af4d (5 weeks ago), 2021-12-16 08:37:33 -0800
    • Engine revision 890a5fca2e
    • Dart version 2.15.1

[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at /Users/blake/Library/Android/sdk
    • Platform android-32, build-tools 32.0.0
    • ANDROID_HOME = /Users/blake/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7249189)

[✓] VS Code (version 1.63.0)
    • VS Code at /Applications/Visual Studio Code.app/Contents
    • Flutter extension version 3.32.0

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 97.0.4692.71

• No issues found!
```

</details>

**Comment by BlakeWuh** [comment]
@flutter-symbolizer-bot [#96850 (comment)](https://github.com/flutter/flutter/issues/96850#issue-1107923709) flutter#v2.8.1 force ios release arm64

**Comment by danagbemava-nc** [comment]
@flutter-symbolizer-bot https://github.com/flutter/flutter/issues/96850#issue-1107923709 force flutter#v2.8.1 ios release arm64

**Comment by danagbemava-nc** [comment]
Hi @BlakeWuh, can we get a list of devices that are affected by this?

**Comment by BlakeWuh** [comment]
Device list
iPhone 7 Plus
iPhone 6s
iPhone 7
iPhone 8 Plus
iPhone X
iPhone 8
iPhone X
iPhone 7 Plus
iPhone 8 Plus

OS list
iOS 14, 15

BTW, our app don't support iOS 12 or below

**Comment by danagbemava-nc** [comment]
Keeping it open for further insights from the team. 

It seems to be related to the `AccessibilityBridge`.

**Comment by HansMuller** [comment]
CC @chunhtai

**Comment by maheshmnj** [comment]
Probably related to https://github.com/flutter/flutter/issues/96851 which has been fixed on master

**Comment by chunhtai** [comment]
Should be fixed in https://github.com/flutter/engine/pull/28495 which should be in 2.10.1

**Comment by chinmaygarde** [comment]
Closing as fixed.
